cli/command/user/update_test.go

```diff
@@ -63,3 +66,108 @@ func TestProcessGroups(t *testing.T) {
 		})
 	}
 }
+
+func TestVerifyUpdate(t *testing.T) {
+	testcases := []struct {
+		name       string
+		updateOpts updateOptions
+		wantErr    error
+	}{
+		{
+			name: "invalid username",
+			updateOpts: updateOptions{
+				username: "%$#",
+			},
+			wantErr: errors.New(`Username doesn't follow format "[a-zA-Z0-9]+"`),
+		},
+		{
+			name: "valid username",
+			updateOpts: updateOptions{
+				username: "foo",
+			},
+		},
+		{
+			name: "empty username",
+			updateOpts: updateOptions{
+				username: "",
+			},
+		},
```

Comment:
I'm doubtful about this case. Shouldn't it invalidate an empty string username? I think update would fail anyway without a username.

Reply:
yeah, I'd expect we would throw an error if username was empty?

Reply:
So the code isn't returning an error on empty username?